pkg/grpc/gateway/request.go

```diff
@@ -154,3 +165,57 @@ func doHTTPRequest(ctx context.Context, req *resty.Request) (interface{}, error)
 		Data:        res.Body(),
 	}, nil
 }
+
+func doHTTPStreamingRequest(ctx context.Context, req *resty.Request) (any, <-chan error, error) {
+	res, err := req.SetContext(ctx).
+		SetDoNotParseResponse(true).
+		Send()
+	if err != nil {
+		return nil, nil, fmt.Errorf("send request: %w", err)
+	}
+	if res.IsError() {
```

Comment:
because we are don't parse response `res.Error` will be empty, we need to do that same as what we do in `DoStreamingRequest`
```
	if rawRes.IsError() {
		body := rawRes.RawBody()
		defer func() { _ = body.Close() }()
		data, err := io.ReadAll(body)
		if err != nil {
			return nil, nil, fmt.Errorf("read error response body: %w", err)
		}

		var res streamingResponse
		if err := json.Unmarshal(data, &res); err != nil {
			return nil, nil, fmt.Errorf("unmarshal raw response: %w", err)
		}
		rawErrRes, ok := res[streamingResponseErrorKey]
		if !ok {
			return nil, nil, errors.New(string(data))
		}
		var errResp rpcstatus.Status
		if err := c.Unmarshal(rawErrRes, &errResp); err != nil {
			return nil, nil, fmt.Errorf("unmarshal error response: %w", err)
		}
		if err := status.ErrorProto(&errResp); err != nil {
			return nil, nil, err
		}
		return nil, nil, status.Error(HTTPStatusToCode(rawRes.StatusCode()), rawRes.String())
	}
```